crates/iceberg-rest/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "iceberg-rest"
```

Comment:
How about calling it `iceberg-catalog-rest`?

Reply:
I leave it as `iceberg-rest` on purpose. Maybe in future we can build rest catalog server in rust.

Reply:
I am considering `iceberg-catalog-hive` and `iceberg-catalog-jdbc`. If we follow the same naming style as `iceberg-rest`, it might be a bit confusing. For instance, `iceberg-hive` and `iceberg-jdbc`.

Reply:
Make sense. The rest api is mostly about catalog operations, so adding `catalog` prefix is not confusing.